pkg/util/test/util.go

```diff
@@ -116,7 +116,7 @@ var TestSpecPath = "testspec"
 var (
 	pxVer2_12, _  = version.NewVersion("2.12.0")
 	opVer1_10, _  = version.NewVersion("1.10.0")
-	opVer1_9_1, _ = version.NewVersion("1.9.1")
+	opVer1_9_1, _ = version.NewVersion("1.9.1-")
```

Comment:
1.9.1-dev is greater than this. but less than 1.9.1

Reply:
Are you sure we need that `-`? I am pretty sure we can leave it as is, especially once 1.9.1 gets released

Reply:
yes look at latest passed test https://jenkins.pwx.dev.purestorage.com/view/Newstack%20Dashboard/job/Operator/job/operator-integration-basic/344/consoleFull
```
09:56:31 time="2022-08-23T16:56:31Z" level=info msg="Testing portworx-operator version: 1.9.1-dev"
09:56:31 time="2022-08-23T16:56:31Z" level=info msg="Image inside stork-f88f5ddcf-bnphq[stork] matches, expected: openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
09:56:31 time="2022-08-23T16:56:31Z" level=info msg="Image inside stork-f88f5ddcf-fpvjm[stork] matches, expected: openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
09:56:31 time="2022-08-23T16:56:31Z" level=info msg="Image inside stork-f88f5ddcf-sg572[stork] matches, expected: openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
```
this is wrong. after the change
```
18:22:20 time="2022-08-24T01:22:21Z" level=info msg="Testing portworx-operator version: 1.9.1-dev"
18:22:20 time="2022-08-24T01:22:21Z" level=info msg="Image inside stork-f88f5ddcf-7jlxq[stork] matches, expected: docker.io/openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
18:22:20 time="2022-08-24T01:22:21Z" level=info msg="Image inside stork-f88f5ddcf-glfpc[stork] matches, expected: docker.io/openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
18:22:20 time="2022-08-24T01:22:21Z" level=info msg="Image inside stork-f88f5ddcf-q9lh4[stork] matches, expected: docker.io/openstorage/stork:2.11.2, actual: docker.io/openstorage/stork:2.11.2"
```
we don't get error if the image validation actually fails.